x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/alerts/alerts_card.tsx

```diff
@@ -39,7 +51,11 @@ export const AlertsCard: OnboardingCardComponent = ({
         alignItems="flexStart"
       >
         <EuiFlexItem grow={false}>
-          <EuiText data-test-subj="alertsCardDescription" size="s" color="subdued">
+          <EuiText
+            data-test-subj="alertsCardDescription"
+            size="s"
+            color={isDarkMode ? '' : 'subdued'}
```

Comment:
Can we be more explicit:
```suggestion
            color={isDarkMode ? 'text' : 'subdued'}
```